src/main/java/com/technonext/ota/b2c/tour/controller/PackageCategoryController.java

```diff
@@ -26,8 +26,14 @@ public PackageCategoryController(PackageCategoryService packageCategoryService)
     }
 
     @GetMapping("/search")
-    @Operation(summary = "API Endpoints for Package Category searching")
-    public ResponseEntity<List<PackageCategoryResponse>> search(){
-        return new ResponseEntity<>(packageCategoryService.getAllByInactiveHajjUmrah(), HttpStatus.OK);
+    @Operation(summary = "API Endpoints for Inactive HajjUmrah Package Category")
```

Comment:
correct the summary text: its not inactive HajjUmrah Package. it will be Tour Package Category